src/test/fuzz/FuzzTransaction.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "test/fuzz/FuzzTransaction.h"
+#include "crypto/SignerKey.h"
+#include "invariant/InvariantManager.h"
+#include "ledger/LedgerTxn.h"
+#include "main/Application.h"
+#include "test/TestUtils.h"
+#include "transactions/TransactionUtils.h"
+
+using namespace stellar;
+
+namespace stellar
+{
+namespace fuzztest
+{
+
+void
+attemptTx(TransactionFramePtr tx, Application& app)
+{
+    LedgerTxn ltx(app.getLedgerTxnRoot());
+    ++ltx.loadHeader().current().ledgerSeq;
+
+    TransactionResultCode code;
+    {
+        LedgerTxn ltxFeeProc(ltx);
+        tx->checkValid(ltxFeeProc, 0);
+        code = tx->getResult().result.code();
+        if (code != txNO_ACCOUNT)
+        {
+            auto baseFee = ltxFeeProc.loadHeader().current().baseFee;
+            tx->processFeeSeqNum(ltxFeeProc, baseFee);
+        }
+        ltxFeeProc.commit();
+    }
+
+    try
+    {
+        LedgerTxn ltxTx(ltx);
+        tx->apply(app, ltxTx);
+        // after application, clear cache so that they don't persist between
+        // executions and confuse the fuzzer, causing it to think some Tx with
+        // [op0...opN] can result in multiple different execution paths.
+        app.getLedgerTxnRoot().clearBestOfferAndEntryCaches();
```

Comment:
I thought the `reset` code would be done *before* running a test (at the same time than reseeding)